recipes/mumps/Makefile.debian.SEQ_mac

```diff
@@ -0,0 +1,58 @@
+#  This file is part of MUMPS 5.0.1, released
+#  on Thu Jul 23 17:08:29 UTC 2015
+#
+# These settings for a PC under Debian/linux with standard packages :
+# metis (parmetis), scotch (ptscotch), openmpi, gfortran
+
+# packages installation :
+# apt-get install libmetis-dev libparmetis-dev libscotch-dev libptscotch-dev libatlas-base-dev openmpi-bin libopenmpi-dev
+
+# Begin orderings
+LSCOTCH   = -L$(PREFIX)/lib -lesmumps -lscotch -lscotcherr
+ISCOTCH   = -I$(PREFIX)/include/
+
+LPORDDIR = $(topdir)/PORD/lib/
+IPORD    = -I$(topdir)/PORD/include/
+LPORD    = -L$(LPORDDIR) -lpord
+
+IMETIS    = -I$(PREFIX)/include/
+LMETIS    = -L$(PREFIX)/lib -lmetis
+
+# Corresponding variables reused later
+ORDERINGSF = -Dmetis -Dpord -Dscotch
+ORDERINGSC  = $(ORDERINGSF)
+LORDERINGS = $(LMETIS) $(LPORD) $(LSCOTCH)
+IORDERINGSF = $(ISCOTCH)
+IORDERINGSC = $(IMETIS) $(IPORD) $(ISCOTCH)
+# End orderings
+################################################################################
+
+PLAT    =
+LIBEXT  = .a
+OUTC    = -o
+OUTF    = -o
+RM = /bin/rm -f
+CC = gcc
+FC = gfortran
+FL = gfortran
```

Comment:
Actually, since I changed `gcc` to `clang` for OS X. So, the files won't be the same, therefore I don't add `fPIC` here. Agree?